ingestion_server/test/unit_test.py

```diff
@@ -140,9 +140,9 @@ def test_url_protocol_fix():
         result = CleanupFunctions.cleanup_url(bad_url, tls_support_cache)
         expected = "'https://flickr.com'"
 
-        bad_http = "neverssl.com"
+        bad_http = "baidu.com"
         result_http = CleanupFunctions.cleanup_url(bad_http, tls_support_cache)
-        expected_http = "'http://neverssl.com'"
+        expected_http = "'http://baidu.com'"
```

Comment:
Was this change intentional?

Reply:
NeverSSL now supports SSL ☹️ I read that Baidu doesn't. The tests are passing locally, not sure why they're failing on CI.

Reply:
Do these make an actual HTTP request during the tests?

If so, depending on the behavior of a 3rd party server for the validity of our test suite is not good. If we need to test a non-SSL http request, then we should mock the request using something like VCR or nock (whatever the Python equivalent of those is).

Reply:
Updated the test to use mocks in 2181e32.